admin/build.sbt

```diff
@@ -1,12 +1,19 @@
 enablePlugins(PlayScala)
 enablePlugins(SystemdPlugin)
 
+import play.sbt.routes.RoutesKeys
+
 bashScriptExtraDefines ++= Seq(
   s"addJava -Dorg.quartz.properties=${defaultLinuxConfigLocation.value}/${(Linux / packageName).value}/quartz.properties",
   "addJava -Dpidfile.path=/run/piezo-admin/piezo-admin.pid",
   s"addJava -Dhttp.port=${PlayKeys.playDefaultPort.value}",
 )
 
+// Workaround for https://github.com/playframework/playframework/issues/7382
+// so we don't get unused import warnings
+RoutesKeys.routesImport := Seq.empty
```

Comment:
we could instead just silence warnings for routes, down where we do so for twirl files

Reply:
We could. This feels cleaner to me though.